app/presenters/search_presenters.py

```diff
@@ -205,17 +212,253 @@ def _add_highlighting(self):
                         ''.join(service['highlight']['serviceSummary'])
                     )
 
-    def _get_search_summary(self, results_total):
-        template = u"<span class='search-summary-count'>{}</span> {} found"
-        if int(results_total) == 1:
-            return Markup(template.format('1', u"result"))
-        else:
-            return Markup(template.format(results_total, u"results"))
-
     def __init__(self, response):
         self.search_results = response['services']
         self._add_highlighting()
-        self.summary = self._get_search_summary(response['meta']['total'])
         self.total = response['meta']['total']
         if 'page' in response['meta']['query']:
             self.page = response['meta']['query']['page']
+
+
+class SearchSummary(object):
+    """Provides a paragraph summarising the search performed and results"""
+
+    COUNT_PRE_TAG = '<span class="search-summary-count">'
+    COUNT_POST_TAG = '</span>'
+    KEYWORDS_PRE_TAG = '<em>'
+    KEYWORDS_POST_TAG = '</em>'
+    LOT_PRE_TAG = '<em>'
+    LOT_POST_TAG = '</em>'
+
+    @staticmethod
+    def write_parts_as_sentence(parts):
+        sentence = [part for part in parts if part is not None]
+        if len(sentence) > 0:
+            return u" ".join(sentence)
+
+    @staticmethod
+    def write_list_as_sentence(input_list, final_conjunction):
+        if len(input_list) == 1:
+            return u"{}".format(input_list[0])
+        else:
+            start = input_list[0:-1]
+            end = input_list[-1]
+            formatted_conjunction = " {} ".format(final_conjunction)
+            return formatted_conjunction.join([u', '.join(start), end])
+
+    def __init__(self, results_total, request_args, filter_groups):
+        self._set_initial_sentence(results_total, request_args)
+        self.filter_groups = self._group_request_filters(
+            request_args,
+            filter_groups
+        )
+        self.filters_fragments = []
+        SummaryRules.load_rules()
+        for group in self.filter_groups:
+            group_id = group[0]
+            filters = group[1]
+            group_rules = SummaryRules(group_id)
+            if group_rules.exist is True:
+                self.filters_fragments.append(
+                    SummaryFragment(
+                        group_id=group_id,
+                        filters=filters,
+                        rules=group_rules)
+                )
+
+    def _set_initial_sentence(self, results_total, request_args):
+        template = u"{} found containing {} in {}"
+        keywords = u"{}&ldquo;{}&rdquo;{}".format(
```

Comment:
Agreed.  Might also need something to change quotes into fancy quotes for phrase search strings, as they come through as plain quotes at the moment:
![screen shot 2015-06-08 at 11 23 39](https://cloud.githubusercontent.com/assets/6525554/8032791/fba462a6-0dd0-11e5-86e1-c697ff1ca050.png)